README.md

```diff
@@ -10,37 +10,46 @@ This repo contains code for interacting with the OSF APIv2 inside of an Ember ap
 
 Please read the [CONTRIBUTING.md](https://github.com/CenterForOpenScience/ember-osf/blob/develop/.github/CONTRIBUTING.md)
 
+## Installation (for Development)
+
+* `git clone` this repository
+* `npm install`
+* `bower install`
+
 ## Using this code in an Ember app
 
 1. Clone the repository: `git clone https://github.com/CenterForOpenScience/ember-osf.git`
 2. From the consuming Ember app:
   - install the addon and it's dependencies: `ember install ../ember-osf`
+	- this generates a config/local.yml file (see 'Configuration' below)
   - link the app for local development: `npm link ../ember-osf`
-  - generate a settings file (see 'Configuration' below)
   - Import code from ember-osf like:
   ```javascript
   import Ember from 'ember';
   import OsfLoginRouteMixin from 'ember-osf/mixins/osf-login-route';
 
   export default Ember.Route.extend(OsfLoginRouteMixin);
   ```
-  
-#### Ember Data: Using the OSF models
-  
-The models, serializers, adapters bundled in this addon with be available to you automatically. 
-For example, simply do:
-```javascript
-this.store.findAll('node')
-```
-to fetch all nodes.
 
-## Installation (for Development)
+## Configuration
 
-* `git clone` this repository
-* `npm install`
-* `bower install`
+#### local.yml settings
 
-## Configuration
+This file is structured like:
+```yaml
+<backend>:
+  CLIENT_ID: null
+  PERSONAL_ACCESS_TOKEN: null
+  OAUTH_SCOPES: osf.full_read osf.full_write
+```
+
+You will need to fill out options for each backend you want to use (see 'Running' below).
+We reccomend using the 'test' backend for development and testing as it is the most stable
+of our staging enviorments.
```

Comment:
Language notes:

- Spell check
  - Typo: should be `recommend`
  - Typo: should be `environments`

Also, we shouldn't refer to `test` as a staging environment. As far as I know, the original intent was to always mirror master- it's a tool for external integrators, more akin to `demo.dataverse.org` than to a staging environment.